### PR TITLE
Register fields control page

### PR DESCRIPTION
**Pull request by yana617**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -54,4 +53,19 @@ body {
   font-weight: bold;
   padding: 16px;
 }
+
+.modal__wrapper {
+  position: fixed;
+  z-index: 9998;
```
Comment (Aphelmion) [inline src/App.vue:59]: Яна ВСМЫЫЫСМЛЕ, [07.03.21 22:45]
упс

Яна ВСМЫЫЫСМЛЕ, [07.03.21 22:45]
убрать надо

File: src/components/register-fields-control-view/AdditionalField.vue [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+        description: this.descriptionModel,
+      };
+      this.$store.dispatch('additionalFields/updateAdditionalField', body).then(() => {
+        this.$store.dispatch('additionalFields/getAdditionalFields');
+        this.edit = false;
+      });
+    },
+    cancel() {
+      this.labelModel = this.label;
+      // this.iconModel = this.icon;
```
Comment (Aphelmion) [inline src/components/register-fields-control-view/AdditionalField.vue:70]: commented code is needed?

File: src/components/register-fields-control-view/AdditionalField.vue [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+      formData.append('icon', icon);
+      this.$store.dispatch('additionalFields/uploadIcon', formData).then(() => {
+        const newIcon = this.uploadedIcon;
+        const updatedField = {
+          id: this.id,
+          icon: newIcon,
+          label: this.labelModel,
+          description: this.descriptionModel,
+        };
+        console.log(this.id);
```
Comment (Aphelmion) [inline src/components/register-fields-control-view/AdditionalField.vue:91]: remove pls

File: src/views/AdminPanel.vue [hunk 4]
```diff
@@ -77,7 +73,7 @@ $headerHeight: 50px;
       text-decoration: none;
 
       &:hover {
-        color:#ccc;
+        color: #ccc;
```
Comment (Aphelmion) [inline src/views/AdminPanel.vue:76]: may be it is better to have same constants like LIGHT_GRAY  = #ccc and than import it and use in such code

File: src/views/RegisterFieldsControl.vue [hunk 5]
```diff
@@ -1,12 +1,158 @@
...
+
+  &__field {
+    &__title {
+      color: black;
+      font-size: 16px;
+      margin-top: 8px;
+    }
+    &__description {
+      font-size: 12px;
+      color: #999;
```
Comment (Aphelmion) [inline src/views/RegisterFieldsControl.vue:133]: another gray?

File: src/components/register-fields-control-view/AdditionalField.vue (hunk 2 above)
Comment (yana617) [reply src/components/register-fields-control-view/AdditionalField.vue:70]: yes

File: src/views/AdminPanel.vue (hunk 4 above)
Comment (yana617) [reply src/views/AdminPanel.vue:76]: It's hard. (discussed)

File: src/views/RegisterFieldsControl.vue (hunk 5 above)
Comment (yana617) [reply src/views/RegisterFieldsControl.vue:133]: yes, gave this color a name

File: src/App.vue (hunk 1 above)
Comment (yana617) [reply src/App.vue:59]: fixed to lower value

**Review by Aphelmion — APPROVED** [review]

**Pull request merged**